source/common/common/utility.cc

```diff
@@ -196,4 +196,11 @@ const std::string& StringUtil::nonEmptyStringOrDefault(const std::string& s,
   return s.empty() ? default_value : s;
 }
 
+std::string StringUtil::toUpper(const std::string& s) {
```

Comment:
I don't see this used? Needed in future? If so, fine, but nit can we add some specific tests for this?